spoor/toolchains/xcode/swiftc.py

```diff
@@ -16,6 +16,11 @@
 def main(argv, build_tools):
   args = argv[1:]
 
+  # Version check
+  if args[0] == "-v" and len(args) == 1:
```

Comment:
Checking the length before the value at the index will prevent a crash with no args.